.changeset/soft-kiwis-swim.md

```diff
@@ -0,0 +1,5 @@
+---
+'@audius/sdk': major
+---
+
+Change AntiAbuseAttestionError to AntiAbuseOracleAttestationError
```

Comment:
That seems like a small change to warrant an entire major bump? But I guess it's technically breaking...